src/styles/blocks/cell.scss

```
@@ -0,0 +1,27 @@
+.calendar__cell {
+  display: flex;
+
+  border: 1px solid #000;
+  height: $cellSize;
+  width: $cellSize;
+  align-items: center;
+  justify-content: center;
+  background-color: $color;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  transition: 0.5s;
```

Don't forget to specify the properties to which a transition effect should be applied
```suggestion
  transition: transform 0.5s, ...;
```